modules/cnv-web-console-overview-dashboard.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * assembly/architecture
+
+[id="web-console-overview-dashboard_{context}"]
+= Overview Dashboard
+
+* *Details* card provides an brief overview of informational cluster details, including:
+** Cluster name
+** Provider
+** Version number of {CNVProductName} Container Platform.
+* *Inventory* card alerts administrators when direct intervention is required to resolve problems.  Information includes:
+** Number of nodes
+** Number of pods
+** Persistent storage volume claims
+** Virtual machines
+** Bare metal hosts in the cluster, listed according to their state.
+
+[NOTE]
+====
+States listed on this card are "healthy," "not healthy," "attempting to self-heal," and "not healthy and unable to recover without administrator assistance."
+====
+
+* *Health* card summarizes the current health of the cluster as a whole. When a problem exists in the cluster, administrators click on the alert on the card to view more information and understand how to correct the problem. Click *See All* to view the status of each subsystem.
+* *Capacity* card charts help administrators understand when additional resources are required in the cluster. The charts contain an inner ring that displays current consumption, while an outer ring displays thresholds configured for the resource, including information about:
+** CPU time
+** Memory allocation
+** Storage consumed
+** Network resources consumed
+* *Utilization* card shows the capacity of various resources over a specified period of time, to help administrators understand the scale and frequency of high resource consumption.
+* *Events* card lists messages related to recent activity in the cluster, such as pod creation or virtual machine migration to another host.
```

Comment:
the word "pod" should be capitalized = "Pod"

(see https://github.com/openshift/openshift-docs/blob/enterprise-4.1/contributing_to_docs/doc_guidelines.adoc#api-object-formatting)

Reply:
Done. Thanks.